tracee/consts.go

```diff
@@ -26,9 +26,11 @@ const (
 type binType uint8
 
 const (
-	modeSystem uint32 = iota
-	modePid
-	modeContainer
+	ModeProcessAll uint32 = iota
```

Comment:
One thing that I noticed is that when I'm now running tracee in container mode (using your PR code, "-t c"), I'm getting events which are not related to containers.
Took me some time to figure this out, but the reason for this is that ModeProcessAll value is 0.
For this reason, when loading the bpf programs, and while we still didn't set the maps (specifically, the config map with  the mode value), the default mode chosen is ModeProcessAll, and possibly non relevant processes can be traced as well...

Lucky for us, the solution is simple - just change the enum to start from 1 (iota + 1), and respectively shift the defines in the C code by one as well